.github/workflows/test-docker.yml

```diff
@@ -0,0 +1,96 @@
+name: Test in docker
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ['ubuntu-22.04']
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.10'
+          - '2.11'
+
+        image:
+          - ubuntu:focal
+          - ubuntu:jammy
+          - debian:buster
+          - debian:bullseye
+
+        # Only 2.11 is available on debian bookworm
+        include:
+          - tarantool: '2.11'
+            image: debian:bookworm
+
+    container:
+      image: ${{ matrix.image }}
+
+    env:
+      TARANTOOL_CACHE_KEY_SUFFIX: -DA-${{ github.run_id }}
+      DEBIAN_FRONTEND: noninteractive
+      TZ: Etc/UTC
+
+    steps:
+      - name: Setup system
+        run: |
+          apt-get -y update
+          apt-get -y upgrade
+          apt-get -y install --no-install-recommends --no-install-suggests \
+            tzdata sudo lsb-release gnupg ca-certificates rsync
+
+      - name: Setup nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.18.0'
+
+      - uses: actions/checkout@v4
+
+      - id: get-latest
+        run: |
+          node <<'SCRIPT'
+            process.env["INPUT_TARANTOOL-VERSION"] = "${{ matrix.tarantool }}"
+            require("./dist/main").latest_version("${{ matrix.tarantool }}").then(v => {
+              console.log(v)
+              require("./dist/main").setActionOutput('version', v);
+            })
+          SCRIPT
+
+      - name: Setup from scratch
+        uses: ./
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      - name: Check precise version
+        run: |
+          dpkg -s tarantool | grep '^Version: ${{ steps.get-latest.outputs.version }}'
+          # It'll also fail if tarantool is installed from cache but not from apt-get
+
+      - name: Uninstall tarantool
+        run: sudo apt-get -y remove tarantool tarantool-dev tarantool-common
+
+      - name: Setup from cache
+        uses: ./
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      - name: Verify install from cache
+        run: |
+          # Fail if tarantool is installed from apt-get
+          if dpkg -s tarantool; then
+            echo "Tarantool wasn't restored from cache"
+            exit 1
+          fi
+
+      - name: Check branch version
+        run: |
+          T=$(tarantool -e 'print(_TARANTOOL:match("%d+%.%d+")); os.exit()')
+          if [ "$T" != "${{ matrix.tarantool }}" ]; then
+            echo "Tarantool version is $T, expected ${{ matrix.tarantool }}"
+            exit 1
+          else
+            echo "Tarantool version is $T, as expected"
+          fi
```

Comment:
Nit: It can be replaced with the `verify-version` action to reduce code duplication.

https://github.com/tarantool/setup-tarantool/blob/615e8f4e4dfbf4e41631a72660d27f1ede102aa0/.github/workflows/test.yml#L148-L151

Reply:
Fixed.